### PR TITLE
Add Username to Devs.json and implement loading image through github

### PR DESCRIPTION
**Pull request by Eastly1997**
Added "username" to devs.json 
edited known username to minimize editing of devs.json
Load image to a container with circular with Image.Network

issue 
https://github.com/flutterph/devs/issues/9

### CODE REVIEW

File: assets/data/devs.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   {
     "name": "Joshua",
     "roles": ["Software Engineer", "Flutter and Dart GDE"],
-    "about": "Lorem ipsum"
+    "about": "Lorem ipsum",
```
Comment (Jansalvador1445) [inline assets/data/devs.json:5]: I think `username` can be added by the user/contributors by it self. what do you think? Like another issue.

File: lib/features/devboard/devs/devs_list.dart [hunk 2]
```diff
@@ -56,7 +56,13 @@ class _DevListItem extends StatelessWidget {
           Container(
             height: 220,
             width: 220,
-            color: Colors.black,
+            decoration: BoxDecoration(
+              shape: BoxShape.circle,
+              image: DecorationImage(
+                image: NetworkImage("https://avatars.githubusercontent.com/${dev.username}"),
```
Comment (Jansalvador1445) [inline lib/features/devboard/devs/devs_list.dart:62]: Maybe, for the string, can we add a global variable?

Comment (Jansalvador1445) [reply lib/features/devboard/devs/devs_list.dart:62]: e.g. "${variable_name}${dev.username}"

File: assets/data/devs.json (hunk 1 above)
Comment (Jansalvador1445) [reply assets/data/devs.json:5]: and what's the possible issue when there's no `username` on the JSON file?

Comment (Jansalvador1445) [reply assets/data/devs.json:5]: and maybe let's updates the `README.MD` for adding the `username` for the JSON file.

Comment (Eastly1997) [reply assets/data/devs.json:5]: I think so, also we add key for other platform for social media like mentioned in issue https://github.com/flutterph/devs/issues/19

**Comment by Jansalvador1445** [comment]
This is good, merging it now.

**Pull request merged**